velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -46,14 +46,12 @@ enum UTF8CharList {
 /// The `fuzz(type)` method provides the highest degree of entropy. It randomly
 /// generates different types of (possibly nested) encodings given the input
 /// type, including constants, dictionaries, sliced vectors, and more. It
-/// accepts any primitive, complex, or nested types. Additionally,
-/// 'opt.allowLazyVector' can be set to true to enable a chance of generating a
-/// lazy vector:
+/// accepts any primitive, complex, or nested types:
```

Comment:
This is unrelated; just realized we changed the API but hadn't cleaned up the comments